tests/testthat/test-summary-log.R

```diff
@@ -0,0 +1,97 @@
+context("Test creating summary logs")
+
+if (Sys.getenv("METWORX_VERSION") == "" && Sys.getenv("DRONE") != "true") {
+  skip("test-summary only runs on Metworx or Drone")
+}
+
+# references
+OFV_REF <- 2636.846
+PARAM_COUNT_REF <- 7
+
+# helper to run expectations
+test_sum_df <- function(sum_df, .paths, .col_count) {
```

Comment:
This is an unusual pattern. It looks like it saves some code, but it's more difficult to understand what is being tested and how. Let's discuss.

Reply:
yea, it saves a fair amount of code. If you really want to talk it over we can, but I like it.

And honestly, I think it's easier to understand than putting 8 expectations in each test and making the dev look through them to see if they're the same each time. If there's another different option than doing that, then I'm happy to discuss.